README.md

```diff
@@ -1,27 +1,22 @@
-# The Rust Programming Language
+<div align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg" width="512" height="256">
 
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+[Website](https://www.rust-lang.org/) | [Getting started](https://www.rust-lang.org/learn/get-started) | [Learn](https://www.rust-lang.org/learn) | [Documentation](https://www.rust-lang.org/learn#learn-use) | [Contributing](CONTRIBUTING.md)
```

Comment:
That's better!

I think this should either have multiple lines still, separated by the pipe char, or move the links into the "link block" at the end of this section. Whatever you think would be nicer for the inherent split of visualizing the result + diffs, ultimately.

Reply:
Since we are already using "link block" in other parts of the README.md, I decided to use those (and did the same for the tools).